src/main/java/ch/njol/skript/lang/function/Functions.java

```diff
@@ -391,12 +411,24 @@ public static void clearFunctions() {
 		toValidate.clear();
 	}
 
+	/**
+	 * @deprecated Use {@link #getFunctions()}.
+	 */
 	@SuppressWarnings({"unchecked"})
+	@Deprecated(forRemoval = true, since = "INSERT VERSION")
 	public static Collection<JavaFunction<?>> getJavaFunctions() {
 		// We know there are only Java functions in that namespace
 		return (Collection<JavaFunction<?>>) (Object) javaNamespace.getFunctions();
 	}
 
+	/**
+	 * Returns all functions.
+	 * @return All functions.
+	 */
+	public static Collection<Function<?>> getFunctions() {
```

Comment:
I'm feeling a bit mixed on this name. Maybe it could be `getDefaultFunctions()`? I just feel like I might expect a method named like this to also return functions registered in scripts